platform/windows/detect.py

```diff
@@ -264,7 +264,7 @@ def configure(env):
 
         if env['use_lto']:
             env.Append(CCFLAGS=['-flto'])
-            if not env['use_llvm'] and env.GetOption("num_jobs") > 1:
+            if not 'use_llvm' in env and env.GetOption("num_jobs") > 1:
```

Comment:
`use_llvm` just does not make sense here, it's a wrong copy paste from the x11 file.
It should simply be:
```
env.Append(LINKFLAGS=['-flto=' + str(env.GetOption("num_jobs"))])
```
in place of the whole `if`. (provided `-flto=1` works)

Reply:
@akien-mga If my Google-fu didn't fail me, GCC only supports "-flto", not "-flto=<num>"? In which case, why not leave it as it is, as it should now work? And will continue working if yout decide to add support for LLVM on Windows some day. Or only use `env.Append(LINKFLAGS=['-flto'])` I guess.

Reply:
`-flto=1` seems to work on GCC.